app/controllers/plans_controller.rb

```diff
@@ -43,4 +43,21 @@ def search
       # TODO
     end
   end
+
+  def set_autofinger_subscription
+    owner = current_account
+    interest = Account.find_by_username(params[:id])
+    autofinger = owner.interests_in_others.find_or_create_by_interest(interest.id)
+    success = autofinger.update_attributes(priority: params[:priority])
+    if success
+      if params[:priority] == '0'
+        notice = 'User was removed from your autoread list.'
+      else
+        notice = "User is now on your autoread list with priority level of #{params[:priority]}."
+      end
+    else
+      notice = "Could not change autoread priority. Please try again."
```

Comment:
Changed it to "Could not change autoread priority. If this happens more than once, contact the Plans admins at grinnellplans@gmail.com." in 0b1778a19f13ca87be10452d02ffb703d5e70469. How does that sound? I didn't know whether @iangreenleaf wanted 'seek help' in there verbatim :).